packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -185,6 +312,85 @@ export const setupRDSInstanceAndData = async (
   return dbConfig;
 };
 
+/**
+ * Creates a new RDS Aurora serverless V2 cluster with one DB instance using the given input configuration, runs the given queries and returns the details of the created RDS
+ * instance.
+ * @param config Configuration of the database cluster
+ * @param queries Initial queries to be executed
+ * @returns Endpoint address, port and database name of the created RDS cluster.
+ */
+export const setupRDSClusterAndData = async (
+  config: RDSConfig,
+  queries?: string[],
+): Promise<{
+  endpoint: string;
+  port: number;
+  dbName: string;
+  dbInstance: DBInstance;
+  password: string;
+  secretArn: string;
+  managedSecretArn: string;
+}> => {
+  console.log(`Creating RDS ${config.engine} DB cluster with identifier ${config.identifier}`);
+
+  const dbCluster = await createRDSCluster(config);
+
+  const { secretArn } = await storeDbConnectionConfigWithSecretsManager({
+    region: config.region,
+    username: config.username,
+    password: dbCluster.password,
+    secretName: `${config.identifier}-secret`,
+  });
+
+  if (!secretArn) {
+    throw new Error('Failed to store db connection config in secrets manager');
+  }
+
+  const client = new RDSDataClient({ region: config.region });
+
+  // create a new test database with given name
+  const createDBInput: ExecuteStatementCommandInput = {
+    resourceArn: dbCluster.clusterArn,
+    secretArn,
+    sql: `create database ${config.dbname};`,
```

Comment:
Rework this to include dbname as a parameter, to avoid potential SQL injection if dbname is ever compromised. (Plus, it's a good habit to get into to not interpolate into SQL if you can avoid it). In this case, the risk is minimal since we generate the dbname, but if in future we decided "hey, let's let dbname be specified by an environment variable", then we'd be open to executing SQL with untrusted input.